evm/src/executor/mod.rs

```diff
@@ -385,26 +295,11 @@ impl Executor {
         let calldata = Bytes::from(encode_function_data(&func, args)?.to_vec());
 
         // execute the call
-        let mut inspector = self.inspector_config.stack();
-        let stipend = calc_stipend(&calldata, self.env.cfg.spec_id);
-        let mut env = self.build_test_env(from, TransactTo::Call(test_contract), calldata, value);
-        let (ExecutionResult { exit_reason, out, gas_used, gas_refunded, logs }, state_changeset) =
-            self.backend_mut().inspect_ref(&mut env, &mut inspector);
+        let env = self.build_test_env(from, TransactTo::Call(test_contract), calldata, value);
+        let call_result = self.call_raw_with_env(env)?;
 
         // if there are multiple forks we need to merge them
-        let logs = self.backend.merged_logs(logs);
-
-        let executed_call = ExecutedCall {
-            exit_reason,
-            out,
-            gas_used,
-            gas_refunded,
-            state_changeset,
-            logs,
-            stipend,
-            env,
-        };
-        let call_result = convert_executed_call(inspector, executed_call)?;
+        // TODO: not used let logs = self.backend.merged_logs(logs);
```

Comment:
@mattsse currently this call to `merged_logs` does nothing because the `logs` field is omitted in `convert_executed_result`. is that expected behavior and we can remove this call or should it be fixed?

Reply:
ah right, that makes sense, since we get the logs from the log inspector directly, so not required here and can be removed